src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -63,9 +64,17 @@ trait TestDriverComponent extends DriverComponent with DataAccess {
 
   def createTestSubmission(workspace: Workspace, methodConfig: MethodConfiguration, submissionEntity: Entity, rawlsUserRef: RawlsUserRef
                            , workflowEntities: Seq[Entity], inputResolutions: Map[Entity, Seq[SubmissionValidationValue]]
-                           , failedWorkflowEntities: Seq[Entity], failedInputResolutions: Map[Entity, Seq[SubmissionValidationValue]]) = {
+                           , failedWorkflowEntities: Seq[Entity], failedInputResolutions: Map[Entity, Seq[SubmissionValidationValue]]): Submission = {
+    createTestSubmission(workspace, methodConfig, submissionEntity, rawlsUserRef, workflowEntities, inputResolutions, failedWorkflowEntities, failedInputResolutions, WorkflowStatuses.Submitted)
+  }
+
+  def createTestSubmission(workspace: Workspace, methodConfig: MethodConfiguration, submissionEntity: Entity, rawlsUserRef: RawlsUserRef
```

Comment:
These two only seem to differ by the last param so it can be `status: WorkflowStatus = WorkflowStatuses.Submitted`